molecule/README.md

```diff
@@ -4,9 +4,9 @@ This directory contains an example Molecule configuration for an Ansible playboo
 
 The Molecule configuration can be run locally or in a Continuous Integration (CI) environment, to ensure the playbook is always working correctly.
 
-To run the Molecule environment, you must have `molecule` and the `docker` Python library installed:
+To run the Molecule environment, you must have `molecule` and the `molecule-docker` Python library installed:
 
-    pip3 install molecule docker
+    pip3 install molecule molecule-docker
```

Comment:
I typically use the syntax:

```
molecule[docker]
```

Can you update to use that instead?

Reply:
```suggestion
    pip3 install molecule molecule[docker]
```